content/hi/infrastructure-as-code.md

```diff
@@ -0,0 +1,20 @@
+---
+title: इन्फ्रास्ट्रक्चर अस कोड (IaC)
+status: Completed
+category: संकल्पना
+tags: ["इन्फ्रास्ट्रक्चर", "infrastructure", ""]
+---
+
+## यह क्या है 
+
+इन्फ्रास्ट्रक्चर अस कोड में इन्फ्रास्ट्रक्चर की परिभाषा को एक या अधिक फाइलों के रूप में संग्रहीत/स्टोर करने का अभ्यास है। यह पारंपरिक मॉडल की जगह लेता है जहां इन्फ्रास्ट्रक्चर अस कोड को मैन्युअल रूप से प्रावधान किया जाता है, आमतौर पर शेल स्क्रिप्ट या अन्य कॉन्फ़िगरेशन टूल के माध्यम से।
+
+## समस्या
+
+क्लाउड नेटिव तरीके से अनुप्रयोगों का निर्माण करने के लिए इन्फ्रास्ट्रक्चर को डिस्पोजेबल और पुनरुत्पादित करने की आवश्यकता होती है। इसे एक स्वचालित और दोहराने योग्य तरीके से [स्केल](/स्केलेबिलिटी/) ऑन-डिमांड की भी आवश्यकता है, संभावित रूप से मानव हस्तक्षेप के बिना। मैन्युअल प्रावधान [क्लाउड नेटिव एप्लिकेशन](/cloud-native-apps/) की जवाबदेही और पैमाने की आवश्यकताओं को पूरा नहीं कर सकता। मैनुअल इन्फ्रास्ट्रक्चर में परिवर्तन प्रतिलिपि प्रस्तुत करने योग्य नहीं हैं, जल्दी से पैमाने की सीमा में चला जाता है, और गलत कॉन्फ़िगरेशन त्रुटियों का परिचय देता है।
+
+## समाधान 
+
+डेटा सेंटर संसाधनों जैसे सर्वर, लोड बैलेंसर्स और सबनेट को कोड के रूप में प्रस्तुत करके,
+यह इंफ्रास्ट्रक्चर टीम को सभी कॉन्फ़िगरेशन के लिए सत्य का एक ही स्रोत रखने की अनुमति देता है और
+उन्हें [CI](/hi/continuous-integration/)/[CD](/hi/continuous-delivery/) पाइपलाइन में अपने डेटा सेंटर का प्रबंधन करने की भी अनुमति देता है, संस्करण नियंत्रण और परिनियोजन रणनीतियों को लागू करना।
```

Comment:
These sentences seem to start on newlines. We can remove that to make the paragraphs homogenous.